src/content/docs/kubernetes-pixie/kubernetes-integration/installation/install-kubernetes-integration-using-helm.mdx

```diff
@@ -62,76 +59,97 @@ While it is possible to install those charts separately, we strongly recommend u
 - `ksm.enabled=true`: Setting this to `true` will automatically install Kube State Metrics (KSM) for you, which is required for our integration to run. You can set this to false if KSM is already present in your cluster, even if it is on a different namespace.
-- `newrelic-infrastructure.privileged=true`: Can be set to `false` to install a trimmed down version of our integration that does not require extra privileges, such as `hostPath` mounts or running containers as root.
-  - Please note that this will disable detailed process collection from the host.
-  - For performance reasons, our logging solution still requires `hostPath` mounts, regardless of the value of this flag. If this is not allowedin your cluster, you will need to disable the logging solution by specifying `logging.enabled=false`.
 - `prometheus.enabled=true`: Will deploy our Prometheus OpenMetrics integration, which automatically collects data from prometheus endpoints present in the cluster.
-- `webhook.enabled=true`: Will install our minimal webhook, which adds environment variables that, in turn, allows linking applications instrumented with APM with infrastructure data.
+- `webhook.enabled=true`: Will install our minimal webhook, which adds environment variables that, in turn, allows [linking applications instrumented with New Relic APM to Kubernetes](/docs/kubernetes-pixie/kubernetes-integration/link-your-applications/link-your-applications-kubernetes).
 
-Our chart has a comprehensive set of flags and tunables that can be edited to configure our solution to your particular needs. For a full list of all the flags that can be configured, please check [the chart's README](https://github.com/newrelic/helm-charts/tree/master/charts/nri-bundle) and the default [`values.yaml`](https://github.com/newrelic/helm-charts/blob/master/charts/nri-bundle/values.yaml) file.
+Our Kubernetes charts have a comprehensive set of flags and tunables that can be edited to better fit your particular needs. Please check the [Configure the integration](#configure) section below to see what can be changed.
 
-4. Install the Kubernetes integration by running the customized command without `--debug` and `--dry-run`:
+5. Install the Kubernetes integration by running the command without `--debug` and `--dry-run`:
 
-```
-helm upgrade --install <var>newrelic</var> newrelic/nri-bundle \
---namespace <mark>newrelic</mark> --create-namespace \
---set global.licenseKey=<var>YOUR_NEW_RELIC_LICENSE_KEY</var> \
---set global.cluster=<var>K8S_CLUSTER_NAME</var> \
---set ksm.enabled=<mark>true</mark> \
---set newrelic-infrastructure.privileged=<mark>true</mark> \
---set infrastructure.enabled=true \
---set prometheus.enabled=true \
---set webhook.enabled=true \
---set kubeEvents.enabled=true \
---set logging.enabled=true
+```shell
+helm upgrade --install newrelic newrelic/nri-bundle \
+--namespace newrelic --create-namespace \
+-f values-newrelic.yaml \
+--devel
```

Comment:
Would we want to add `--wait --timeout 120`? When the command finish we are completely sure that everything is run & running.

Reply:
I tried to keep this as close to the guided install as possible, and at the same time as short as possible. I think knowledgeable users will still add `--wait` if they like.

Do you think there is a benefit for novice users to use `--wait`?

Reply:
Having `--wait` will lock helm until all the pods enter in `Ready` state and all the probes are working correctly.

I think that right now we do no have any kind of probe like "liveness probe is correct when we connect to the endpoint that we are scrapping" so `--wait` would be almost useless because almost all pods enter in `Ready` state immediately.

 I think I would have both and even add `--wait` to the guided install so we at least have an error that the installation failed instead of "release installed correctly" but then have the pods in CrashLoopBackoff.
 
 This is more a NTH than something mandatory for me to approve this PR (which I consider it wonderful)